phpunit.xml.dist

```diff
@@ -1,15 +1,12 @@
 <?xml version="1.0" encoding="UTF-8"?>
 
-<phpunit backupGlobals="false"
-         backupStaticAttributes="false"
+<phpunit xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:noNamespaceSchemaLocation="http://schema.phpunit.de/5.2/phpunit.xsd"
```

Comment:
why `5.2` ? I'm quite sure it always uses a newer version than that

Reply:
This was copied from symfony/symfony, but I don't think it's required so maybe it's better to remove it (as it was before)

https://github.com/symfony/symfony/blob/abeb8e4ca9adfcc1494df22af9fd9a1c0181b532/phpunit.xml.dist#L4